walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -122,7 +126,68 @@ public void everyAttendeeIsConsidered() {
   }
 
   @Test
-  public void overlappingEvents() {
+  public void query_everyAttendeeIsConsideredExcludingOptionalAttendees() {
+    // Have each person have different events. We should see two options because each person has
+    // split the restricted times.
+    // Events  : |--------------C--------------|
+    //                 |--A--|     |--B--|
+    // Day     : |-----------------------------|
+    // Options : |--1--|     |--2--|     |--3--|
+
+    Collection<Event> events = Arrays.asList(
```

Comment:
nit: might have missed this earlier, but let's try to use ImmutableList.of() instead of Arrays.asList() when we can. Same for the other tests

Reply:
Done, thanks!